libvast/vast/meta_index.hpp

```diff
@@ -73,16 +73,12 @@ class meta_index {
   /// index (in bytes).
   size_t size_bytes() const;
 
-  /// Gets the options for the synopsis factory.
-  /// @returns A reference to the synopsis options.
-  caf::settings& factory_options();
-
   // -- concepts ---------------------------------------------------------------
 
   // Allow debug printing meta_index instances.
   template <class Inspector>
   friend auto inspect(Inspector& f, meta_index& x) {
-    return f(x.synopsis_options_, x.synopses_);
+    return f(x.synopses_);
```

Comment:
I would suggest to insert a dummy `caf::settings` value to avoid the unnecessary breaking.

Reply:
We still serialize this with CAF? 🙄

Reply:
hm, scratch that, we're not writing this to file.

Reply:
Alrighty then.